tests/e2e/suite_test.go

```diff
@@ -33,7 +33,8 @@ import (
 )
 
 const (
-	e2eNamespace = "kubestone-e2e"
+	e2eNamespaceIperf3   = "kubestone-e2e-iperf3"
```

Comment:
I'd recommend specifying the e2eNamespaceIperf3 in iperf3_test.go, and e2eNamespaceSysbench in sysbench_test.go

We can maybe keep the prefix in this file (i.e. `e2eNamespaceBase = 'kubestone-e2e'`) ?

Reply:
I would go with the first option.

Reply:
Done